jobs/delorean/jenkinsfiles/2.0/ews/discovery/umb/Jenkinsfile

```diff
@@ -54,6 +54,52 @@ def createJiraBody(summary, description, gitPrUrl, labels) {
   }"""
 }
 
+String createMailBody(msgInfoName, msgInfoImage, passed, prUrl, jiraIssue){
+    def body = ""
+    body += [
+        "Hi folks,",
+        "-",
+        "We wanted to let you know that we got word of a new build of the `${msgInfoName}` image",
+        "There's some details below about it",
+        "-",
+        "The build that triggered this message: ${msgInfoImage}",
+        "The jenkins pipeline which produced this email: ${env.BUILD_URL}",
+    ].join('\n')
+    if (passed) {
+        if (prUrl != ""){
```

Comment:
Will this ever be null/nil? I think an empty string returns false anyway so you can probably just do if(prUrl)